openfisca_us/variables/gov/states/il/tax/income/personal_exemption/il_is_exemption_eligible.py

```diff
@@ -0,0 +1,21 @@
+from openfisca_us.model_api import *
+
+
+class il_is_exemption_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Whether this tax unit is eligible for any exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = ""
```

Comment:
Also note you'll need state code in yaml tests when using defined_for
```suggestion
    reference = ""
    defined_for = StateCode.IL
```